LVT/Program.cs

```diff
@@ -1,12 +1,24 @@
-﻿using System;
+﻿using LVT.LVT.Services;
+using System;
+using System.IO;
 
 namespace LVT
 {
-    class Program
+    public class Program
     {
+
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            //this takes an absolute path and needs to be adapted for each computer depending on where the file lives that you want to open - but this allows you to pass different template trees.
+            //see Tests for an alternative way that uses project resources instead - so tests can be run no matter on whose environment
+            StreamReader file = File.OpenText(@"C:\Users\beckerfs\Documents\Projects\LVT\Lean-Value-Tree-Visualisation\LVT\SingleBranchLVT.json");
```

Comment:
Noted that this should be expanded to take a file, however as discussed this can be added later.